ztp/siteconfig-generator/siteConfig/siteConfigBuilder.go

```diff
@@ -285,6 +303,16 @@ func (scbuilder *SiteConfigBuilder) getClusterCR(clusterId int, siteConfigTemp S
 				return mapIntf, err
 			}
 			mapIntf[k] = value
+		} else if reflect.ValueOf(v).Kind() == reflect.Slice {
```

Comment:
Can we get rid of else if with switch blocks for better readability?

Reply:
okay, done